mirgecom/profiling.py

```diff
@@ -51,15 +52,68 @@ class SingleCallKernelProfile:
     footprint_bytes: int
 
 
+class StatisticsAccumulator:
+    """Class that provides statistical functions for profile results.
+
+    Values are converted to "Giga".
+    """
```

Comment:
* Add `.. automethod` for methods.
* Document `num_values`.

Reply:
Done

Reply:
Doesn't look done. Forgot to push?